callHealthCheck.ts

```diff
@@ -1,32 +1,34 @@
-import axios from 'axios';
```

Comment:
the switch from `'` to `"` seems backwards? The linter/prettify config should prefer `'`

Reply:
Yes, added in the .prettierrc file.